Makefile

```diff
@@ -1228,9 +1229,12 @@ nova_deploy_cleanup: namespace ## cleans up the service instance, Does not affec
 
 .PHONY: mariadb_kuttl_run
 mariadb_kuttl_run: ## runs kuttl tests for the mariadb operator, assumes that everything needed for running the test was deployed beforehand.
-	kubectl-kuttl test --config ${MARIADB_KUTTL_CONF} ${MARIADB_KUTTL_DIR}
+	MARIADB_KUTTL_DIR=${MARIADB_KUTTL_DIR} kubectl-kuttl test --config ${MARIADB_KUTTL_CONF} ${MARIADB_KUTTL_DIR} --namespace ${NAMESPACE}
```

Comment:
A quick grep does not show any use for `MARIADB_KUTTL_DIR` in the tests, I don't think we need it here

Reply:
You are correct and this is not needed.

I was wondering adding this consistently would be helpful for future test implementation which would require path to some test files, but I can drop this now.